sunpy/net/helio/tests/test_helio.py

```diff
@@ -263,12 +263,18 @@ def test_link_test_on_urlerror(mock_link_test):
 @pytest.fixture(scope="session")
 def client():
     working_links = list(filter(link_test, webservice_parser()))
-    taverna_link = taverna_parser(working_links[0])[0]
+    try:
+        taverna_link = taverna_parser(working_links[0])[0]
+    except IndexError:
+        return None
     return HECClient(taverna_link)
 
 
 @pytest.mark.remote_data
 def test_get_table_names(client):
+    # xfail the test if the server is down
+    if client is None:
```

Comment:
I would rather that it gets xfailed by the figure instead of checking this in the test.

Reply:
sure sir, i will do the required changes

Reply:
Please do not call me sir.